pkg/timestream/macros.go

```diff
@@ -62,17 +63,19 @@ func Interpolate(query models.QueryModel, settings models.DatasourceSettings) (s
 		txt = strings.ReplaceAll(txt, timeToStr, replacement)
 	}
 
-	if strings.Contains(txt, intervalStr) {
+	if strings.Contains(txt, intervalRawStr) {
+		replacement := fmt.Sprintf("%d", query.Interval.Milliseconds())
+		txt = strings.ReplaceAll(txt, intervalRawStr, replacement)
+	}
+
+	if strings.Contains(txt, intervalStr) || strings.Contains(txt, intervalStrAlias) {
 		replacement := fmt.Sprintf("%dms", query.Interval.Milliseconds())
 		if replacement == "0ms" {
```

Comment:
unrelated-- I wonder why this check was not done for any other condition

Reply:
the whole code base here is quite patchy... let's see how much we can refactor.